stock_by_warehouse_mrp_bom/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Functional Rationale:
+---------------------
+
+Provide a nice widget in the product on the mrp_bom line to see the quantity of product
+by warehouse in order to help you to review such information as fast as possible.
+
+.. image:: http://screenshots.vauxoo.com/nhomar/stock_by_warehouse_sale_1.png
```

Comment:
Hi, It should be great to have the new screenshots here you can use as an example the ones added in the [`stock_by_warehouse_stock`](https://github.com/Vauxoo/addons-vauxoo/blob/12.0/stock_by_warehouse_stock/README.rst),  adding the SS that you put [in the MR comment](https://github.com/Vauxoo/addons-vauxoo/pull/1517#issuecomment-1011591319)